entry_tasks/15_find_unique/main.py

```diff
@@ -0,0 +1,7 @@
+if __name__ == "__main__":
+    numbers = input('Enter some numbers, separated by comma: ').replace(" ", "")
```

Comment:
I'm not fully following why we need to replace " " by "", is there a reason?

Reply:
@mykytapavlov just to clear all the spaces, which are possible from user's input.